### ISSUE TITLE
Add a filter by Advisory Severity

### ISSUE THREAD

**Issue by joris974** [kunalnagarco/action-cve#144]
Could there be an optional parameter to notify only on a given set of Advisory Severity?

```yml
with:
    github-token: ...
    severities: |
      HIGH
      CRITICAL
```

Something like this
```ts
export const filterAlertsByAdvisorySeverity = (
  alerts: Alert[],
  severities: AdvisorySeverity[],
) => {
  if (severities.length === 0) {
    return alerts;
  }
  return alerts.filter((alert) =>
    alert.advisory !== undefined
      ? severities.includes(alert.advisory.severity)
      : true
  );
};
```

**Comment by kunalnagar** [comment]
@joris974 - this sounds great. I saw your PR and I think we can do better by switching to the REST API which allows us to filter by severity instead of client-side filtering.

I'm working on a PR for the same: https://github.com/kunalnagarco/action-cve/pull/151

**Comment by kunalnagar** [comment]
Resolved by #151

**Comment by joris974** [comment]
Looks great. Thank you @kunalnagar
